tools/functional-tester/etcd-tester/tester.go

```diff
@@ -109,34 +109,33 @@ func (tt *tester) doRound(round int) (bool, error) {
 		}
 		plog.Printf("%s recovered failure", tt.logPrefix())
 
-		if tt.cluster.v2Only {
-			plog.Printf("%s succeed!", tt.logPrefix())
-			continue
-		}
-
-		if !tt.consistencyCheck {
-			if err := tt.updateRevision(); err != nil {
-				plog.Warningf("%s functional-tester returning with tt.updateRevision error (%v)", tt.logPrefix(), err)
-				return false, err
-			}
-			continue
-		}
-
 		if err := tt.checkConsistency(); err != nil {
 			plog.Warningf("%s functional-tester returning with tt.checkConsistency error (%v)", tt.logPrefix(), err)
 			return false, err
 		}
+
+		if err := tt.updateRevision(); err != nil {
```

Comment:
`checkConsistency` doesn't update the current revision any more. I think that means the update should happen before the stresser starts again so that the current revision doesn't get ahead of the consistency checking, though? Will fix.